globus_action_provider_tools/flask/helpers.py

```diff
@@ -97,7 +103,9 @@ def check_token(request: Request, checker: TokenChecker) -> AuthState:
     """
     Parses a Flask request to extract its bearer token.
     """
-    access_token = request.headers.get("Authorization", "").strip().lstrip("Bearer ")
+    access_token = request.headers.get("Authorization", "").strip()
+    if access_token.startswith("Bearer "):
+        access_token = access_token[len("Bearer ") :]
```

Comment:
As discussed, I introduced this change because `.lstrip()` doesn't work the way the author intended (`str.remove_prefix()` was introduced in newer Python versions because of this common confusion).

I don't have an opinion on changing this behavior, only that I would prefer to do it in a subsequent PR.

Reply:
Yep, and I think we were in agreement on this. It caught my eye for a different reason from why the original caught yours, but any further change is for a future PR. Making this resolved. 🙂

Reply:
I'm gonna keep this unresolved so that it's visible when the page loads for others. :+1: